scripts/lib/CIME/tests/test_sys_test_scheduler.py

```diff
@@ -14,9 +14,11 @@
 
 
 class TestTestScheduler(base.BaseTestCase):
-    @mock.patch.dict(os.environ, {"CIME_MODEL": "cesm"})
     @mock.patch("time.strftime", return_value="00:00:00")
     def test_chksum(self, strftime):  # pylint: disable=unused-argument
+        if utils.get_model() != "cesm":
```

Comment:
I think the intent of this test was to use mock to have it work everywhere, but that doesn't work because:
```
CIME.utils.CIMEError: ERROR: Makes no sense to have empty read-only file: /home/e3sm-jenkins/jenkins-ws/workspace/mappy_cime/E3SM/ccs_config/machines/config_machines.xml
```
The E3SM config files don't live under ccs_config.

Reply:
Yea that was the intent, this should be fine now.